distributions/aws/test/e2e/resources/custom-resource-templates/katib-experiment-random.yaml

```diff
@@ -8,7 +8,7 @@ metadata:
 spec:
   objective:
     type: maximize
-    goal: 0.99
+    goal: 0.90
```

Comment:
was this change able to make this job complete ?

Reply:
yeah it makes the job complete faster, which reduces testing time